Dockerfile

```diff
@@ -0,0 +1,46 @@
+ARG RUBY_VERSION=2.7.6
+FROM ruby:$RUBY_VERSION-alpine
+
+ENV SPOTLIGHT_ENGINE_PATH /spotlight/engine
+ENV SPOTLIGHT_GEM /spotlight/engine
+ENV RAILS_QUEUE inline
+
+RUN apk --no-cache upgrade && \
+  apk add --no-cache \
+  build-base \
+  curl \
+  gcompat \
+  git \
+  imagemagick \
+  less \
+  libxml2-dev \
+  libxslt-dev \
+  nodejs-current \
+  postgresql-dev \
+  shared-mime-info \
+  sqlite-dev \
+  tini \
+  tzdata \
+  yarn \
+  zip
+
+RUN addgroup --gid 10001 --system spotlight && \
+  adduser --uid 10000 --system \
+  --ingroup spotlight --home /spotlight spotlight
+
+USER spotlight
+RUN gem update bundler
+RUN gem install --no-document rails -v '< 6.1'
```

Comment:
Should we make the rails version an `ARG`?

Reply:
oh yeah good idea